docs/runbooks/DeprecatedRHEL6Vm.md

```diff
@@ -0,0 +1,26 @@
+# DeprecatedRHEL6Vm
+
+## Meaning
+
+This alert fires when a running virtual machine is based on RHEL6 template.
+
+## Impact
+<!--USstart-->
```

Comment:
@fossedihelm no need for the US and DS comments. Please remove.

Reply:
@sradco Do you mean to remove the differentiation between US and DS, leaving only
`RHEL6 virtual machines will not be supported in the next release`?